src/main/java/hudson/plugins/git/util/DefaultBuildChooser.java

```diff
@@ -120,7 +120,7 @@ public DefaultBuildChooser() {
                 verbose(listener, "{0} seems to be a non-branch reference (tag?)");
             }
         }
-        
```

Comment:
Could you remove these whitespace changes from the pull request?

I'm not thrilled with the indentation differences in the files, but we either move to fully automated formatting (from within the maven build script) or we stay with minimized differences from one change to the next.  Adjusting white space a commit at a time is not the way I want to go.

My initial proposal to switch to automated formatting was rejected by other maintainers, so I'm working on the assumption that we need to use "diff minimization" rather than "indentation rationalization".  Refer to [CONTRIBUTING](https://github.com/jenkinsci/git-plugin/blob/master/CONTRIBUTING.md) for more information on the current indentation plan.